airbyte-webapp/src/components/base/Button/types.tsx

```diff
@@ -1,10 +1,22 @@
+import React from "react";
+
+export enum ButtonType {
+  Primary = "primary",
+  Secondary = "secondary",
+  Danger = "danger",
+  LightGrey = "lightGrey",
```

Comment:
`lightGrey` is not a good name because it refers to a color. Either this should be `secondary` and the other `secondary-outline`, or maybe just `light` or `neutral`.